content/src/tnips/tnip_2.md

```diff
@@ -0,0 +1,331 @@
+
+## Security Considerations
+
+<!--
+  All TNIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. For example, include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. TNIP submissions missing the "Security Considerations" section will be rejected. A TNIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
+
+  The current placeholder is acceptable for a draft.
+
+  TODO: Remove this comment before submitting
+-->
+
+### Limited Decentralization (Early Stages)
+
+GSMA full-members hold the exclusive right to validate transactions and participate in consensus. The members are decentralized within the context of Mobile Network Operators (MNOs), but eligible committee members are more limited than other protocols because of this requirement. It is beneficial for the protocol and the network at large for GSMA members to stake and operate multiple nodes. However, it will take time for validator numbers to reach a critical point of diversity where the protocol can tolerate more byzantine nodes without significant consequences.
+
+### Denial of Service (DoS)
+
+Validators must share network information with peers and maintain constant communication. It's also important that [NVVs are permissionless](#permissionless-nvvs) and trustless. As the network continues to grow, validator numbers are expected to be limited in the early stages which increases the effectiveness of flooding validators with bogus requests. Spamming ports dedicated to consensus messages between [CVVs](#committee-voting-validators-cvvs) is especially detrimental to network progress.
+
+Additional consideration is needed to enhance the network's robustness against these types of attacks. Currently, the network uses virtual private networks for committees messages. Redundant proxies may also be effective in preventing this type of attack. MNOs have significant experience mitigating these types of attacks and should provide direction towards refining network security.
+
+### Lazy Validators
+
+Lazy validators might subscribe to a peer's execution results through a public RPC and use the data as their own, as if they computed the output individually. Right now, there is some level of trust that validators won't be lazy and skip execution. Stake and knowledge is a reasonable deterrent in the early stages, but more work is needed to remove this trust assumption as the network grows larger. One idea is to have staked [NVVs](#non-voting-validators-nvvs) randomly sample [CVV](#committee-voting-validators-cvvs) execution results throughout each epoch, but that is outside the scope of this proposal.
```

Comment:
Why not expect NVVs to execute all consensus output not just random samples?  If I ran an NVV that is what I would expect actually, trust but verify- everything.